package.json

```diff
@@ -27,15 +27,19 @@
     "url": "https://github.com/johnnyreilly/fork-ts-checker-notifier-webpack-plugin/issues"
   },
   "dependencies": {
+    "fork-ts-checker-webpack-plugin": "^1.0.0-alpha.0",
```

Comment:
I feel like this should be moved into `peerDependencies` instead. Reasoning is that a user might want a different version of fTCWP, and the user can override fTCNWP's tTCWP version.

Reply:
Cool - will do

Reply:
It's now listed as a `devDependency` of:

```
    "fork-ts-checker-webpack-plugin": "^1.0.0-alpha.1",
```

and a `peerDependency` of:

```
    "fork-ts-checker-webpack-plugin": "*"
```

Sound about right?